### DISCUSSION TITLE
Bunker Defence High Rounds

### DISCUSSION THREAD

**Discussion by SESHZombies** [nzp-team/nzportable#839] [Records and Speedruns]
![Capture](https://github.com/user-attachments/assets/632b2e97-2d00-43b9-885f-972b04401c28)

https://www.youtube.com/watch?v=un5_PE5xEKw

**Comment by MotoLegacy** [comment]
Thanks for sharing, that is a nice achievement for the time investment alone. That ammo box really does make that a streamlined process though x)

Doubted the stability of the game before watching this as well. Good to see. Thanks

**Reply by SESHZombies** [reply]
No worries :)

**Comment by Kingjaxgaming420** [comment]
gg

**Comment by mmkkiinn** [comment]
I am 32 rounds

**Comment by Kingjaxgaming420** [comment]
I'm coming for this soon
